### PR TITLE
fix: remove ending periods from non-sentence list items

### PR DESCRIPTION
**Pull request by de-oz**
### Description

Removed periods from the list items which do not represent complete sentences under the `See also` heading.

### CODE REVIEW

**Comment by hamishwillee** [comment]
I don't have a problem with this, but where is the style guide it is meeting? https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide

**Comment by de-oz** [comment]
> I don't have a problem with this, but where is the style guide it is meeting?
> 
> I.e. in https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide I see mostly instruction to use punctuation. For me consistency is more important than punctuation, but following this guide is most important of all.

This PR was based on consistency rather than the style guide, as usually on MDN list items under the `See also` section do not include periods if they aren't complete sentences and only contain links.  If both including a period and omitting it in such cases are acceptable, then I guess this PR can be closed, but if not, please let me know how this inconsistency should be handled.

**Comment by hamishwillee** [comment]
> please let me know how this inconsistency should be handled.

Of course. @Rumyra Can we have direction on whether see also links bullet lists should have punctuation or not. Then we can update the style guide (or at least know the rule). FWIW I would prefer no punctuation, because often these will be single words or heading names that are not naturally complete sentences.

**Comment by dipikabh** [comment]
Hi @hamishwillee and @de-oz, hopefully the guidelines for 'See also' and the punctuation to be used with the links will soon be out. Stay tuned.

**Comment by hamishwillee** [comment]
Hi @dipikabh, Did final rules get agreed here?

**Comment by teoli2003** [comment]
The short answer is yes. The final PR to get them in the meta docs is waiting for a last round of approval to mark consensus (your approval to show you adhere to the soon-confirmed consensus is welcome): https://github.com/mdn/content/pull/25191

**Comment by hamishwillee** [comment]
Yes, I had a couple of reservations (mostly matters of "taste"), but it is a significant improvement.

Looks like see also are mostly bulleted lists, where items are phrases with punctuation. So this PR would not be accepted. There is some inconsistency between the instructions and representative examples so I'll wait until it merges before proceeding with this.

Thanks!

File: files/en-us/web/api/htmlcanvaselement/getcontext/index.md [hunk 1]
```diff
@@ -188,9 +188,9 @@ Now you have the [2D rendering context](/en-US/docs/Web/API/CanvasRenderingConte
 
 - {{domxref("CanvasRenderingContext2D.getContextAttributes()")}}
 - {{domxref("WebGLRenderingContext.getContextAttributes()")}}
-- The interface defining it, {{domxref("HTMLCanvasElement")}}.
+- The interface defining it, {{domxref("HTMLCanvasElement")}}
```
Comment (dipikabh) [inline files/en-us/web/api/htmlcanvaselement/getcontext/index.md:191]:
```suggestion
- {{domxref("HTMLCanvasElement")}}: Interface used to define the getContext() method
```

File: files/en-us/web/api/htmlcanvaselement/getcontext/index.md [hunk 2]
```diff
@@ -188,9 +188,9 @@ Now you have the [2D rendering context](/en-US/docs/Web/API/CanvasRenderingConte
...
 - {{domxref("OffscreenCanvas.getContext()")}}
 - Available rendering contexts: {{domxref("CanvasRenderingContext2D")}},
   {{domxref("WebGLRenderingContext")}} and {{domxref("WebGL2RenderingContext")}} and
```
Comment (dipikabh) [inline files/en-us/web/api/htmlcanvaselement/getcontext/index.md:194]:
```suggestion
- {{domxref("CanvasRenderingContext2D")}},
  {{domxref("WebGLRenderingContext")}}, {{domxref("WebGL2RenderingContext")}}, {{domxref("ImageBitmapRenderingContext")}}: Available rendering contexts
```

File: files/en-us/web/api/htmlcanvaselement/getcontext/index.md [hunk 3]
```diff
@@ -188,9 +188,9 @@ Now you have the [2D rendering context](/en-US/docs/Web/API/CanvasRenderingConte
...
-  {{domxref("ImageBitmapRenderingContext")}}.
-- Available color spaces: [sRGB color space](https://en.wikipedia.org/wiki/SRGB) and [display-p3 color space](https://en.wikipedia.org/wiki/DCI-P3).
+  {{domxref("ImageBitmapRenderingContext")}}
+- Available color spaces: [sRGB color space](https://en.wikipedia.org/wiki/SRGB) and [display-p3 color space](https://en.wikipedia.org/wiki/DCI-P3)
```
Comment (dipikabh) [inline files/en-us/web/api/htmlcanvaselement/getcontext/index.md:196]:
```suggestion
- [sRGB color space](https://en.wikipedia.org/wiki/SRGB) on Wikipedia
- [display-p3 color space](https://en.wikipedia.org/wiki/DCI-P3) on Wikipedia
```

File: files/en-us/web/api/htmlcanvaselement/height/index.md [hunk 4]
```diff
@@ -50,6 +50,6 @@ console.log(canvas.height); // 300
 
 ## See also
 
-- The interface defining it, {{domxref("HTMLCanvasElement")}}.
+- The interface defining it, {{domxref("HTMLCanvasElement")}}
```
Comment (dipikabh) [inline files/en-us/web/api/htmlcanvaselement/height/index.md:53]:
```suggestion
- {{domxref("HTMLCanvasElement")}}: Interface used to define the `HTMLCanvasElement.height` property
```
would be nice to replace the pronoun 'it' to be more specific

File: files/en-us/web/api/htmlcanvaselement/height/index.md [hunk 5]
```diff
@@ -50,6 +50,6 @@ console.log(canvas.height); // 300
...
 - The other property controlling the size of the canvas,
```
Comment (dipikabh) [inline files/en-us/web/api/htmlcanvaselement/height/index.md:54]:
```suggestion
- {{domxref("HTMLCanvasElement.width")}}: Other property used to control the size of the canvas
```

File: files/en-us/web/api/htmlcanvaselement/height/index.md [hunk 6]
```diff
@@ -50,6 +50,6 @@ console.log(canvas.height); // 300
...
-  {{domxref("HTMLCanvasElement.width")}}.
+  {{domxref("HTMLCanvasElement.width")}}
```
Comment (dipikabh) [inline files/en-us/web/api/htmlcanvaselement/height/index.md:55]:
```suggestion

```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Edited two files to make the list items follow the new ["See also" section guidelines](https://github.com/mdn/content/pull/25191).

In general, the following structure is being followed:
`<link text>: Description phrase(no punctuation)`
The description phrase is optional.

So yes, removing periods here is correct (because they're not complete sentences) but also rephrase the list item so that the link text is at the beginning.

File: files/en-us/web/api/htmlcanvaselement/getcontext/index.md [hunk 7]
```diff
@@ -182,9 +182,9 @@ Now you have the [2D rendering context](/en-US/docs/Web/API/CanvasRenderingConte
...
+- {{domxref("HTMLCanvasElement")}}: Interface used to define the getContext() method
 - {{domxref("OffscreenCanvas.getContext()")}}
-- Available rendering contexts: {{domxref("CanvasRenderingContext2D")}},
-  {{domxref("WebGLRenderingContext")}} and {{domxref("WebGL2RenderingContext")}} and
-  {{domxref("ImageBitmapRenderingContext")}}.
-- Available color spaces: [sRGB color space](https://en.wikipedia.org/wiki/SRGB) and [display-p3 color space](https://en.wikipedia.org/wiki/DCI-P3).
+- {{domxref("CanvasRenderingContext2D")}}, {{domxref("WebGLRenderingContext")}},
+  {{domxref("WebGL2RenderingContext")}}, {{domxref("ImageBitmapRenderingContext")}}: Available rendering contexts
+- [sRGB color space](https://en.wikipedia.org/wiki/SRGB) on Wikipedia
+- [display-p3 color space](https://en.wikipedia.org/wiki/DCI-P3) on Wikipedia
```
Comment (dipikabh) [inline files/en-us/web/api/htmlcanvaselement/getcontext/index.md:190]: There is an issue with the ordering of the items here.

These two should probably be at the top:
- HTMLCanvasElement
- OffscreenCanvas.getContext()

Followed by these two, perhaps as a single item:
- CanvasRenderingContext2D.getContextAttributes(), WebGLRenderingContext.getContextAttributes()

File: files/en-us/web/api/htmlmediaelement/seekable/index.md [hunk 8]
```diff
@@ -35,6 +35,6 @@ for (let count = 0; count < timeRangesObject.length; count++) {
 
 ## See also
 
-- The interface defining it, {{domxref("HTMLMediaElement")}}.
+- {{domxref("HTMLMediaElement")}}: Interface used to define the `HTMLMediaElement.seekable` property
 - [Media Source Extensions API](/en-US/docs/Web/API/Media_Source_Extensions_API)
```
Comment (dipikabh) [inline files/en-us/web/api/htmlmediaelement/seekable/index.md:39]:
```suggestion
- [Media Source API](/en-US/docs/Web/API/Media_Source_Extensions_API)
```

File: files/en-us/web/api/htmlmediaelement/texttracks/index.md [hunk 9]
```diff
@@ -87,19 +87,13 @@ for (const track of tracks) {
...
-- The interface defining it, {{DOMxRef("HTMLMediaElement")}}.
-- {{HTMLElement("video")}} and {{HTMLElement("audio")}}
-- {{DOMxRef("VideoTrack")}} and {{DOMxRef("VideoTrackList")}}
-- {{DOMxRef("AudioTrack")}} and {{DOMxRef("AudioTrackList")}}
-- VideoTrackList
+- {{domxref("HTMLMediaElement")}}: Interface used to define the `HTMLMediaElement.textTracks` property
+- {{HTMLElement("video")}}, {{HTMLElement("audio")}}
+- {{DOMxRef("VideoTrack")}}, {{DOMxRef("VideoTrackList")}}
+- {{DOMxRef("AudioTrack")}}, {{DOMxRef("AudioTrackList")}}
+- [`addtrack`](/en-US/docs/Web/API/VideoTrackList/addtrack_event),
```
Comment (dipikabh) [inline files/en-us/web/api/htmlmediaelement/texttracks/index.md:94]:
```suggestion
- {HTMLElement("audio")}}, {{HTMLElement("video")}}
- {{DOMxRef("AudioTrack")}}, {{DOMxRef("AudioTrackList")}}
- {{DOMxRef("VideoTrack")}}, {{DOMxRef("VideoTrackList")}}
- [`addtrack`](/en-US/docs/Web/API/VideoTrackList/addtrack_event),
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks for the fixes! Suggesting a few more edits, including some untouched list items

File: files/en-us/web/api/htmlmediaelement/canplaytype/index.md [hunk 10]
```diff
@@ -51,7 +51,7 @@ console.log(obj.canPlayType("video/mp4")); // "maybe"
 
 ## See also
 
-- The interface defining it, {{domxref("HTMLMediaElement")}}.
+- {{domxref("HTMLMediaElement")}}: Interface used to define the `HTMLMediaElement.canPlayType()` method
 - {{domxref("MediaCapabilities")}} in the Media Capabilities API
 - [Handling media support issues in web content](/en-US/docs/Web/Media/Formats/Support_issues)
 - [Guide to media types and formats on the web](/en-US/docs/Web/Media/Formats)
```
[the remark this answers is not in the document]
Comment (de-oz) [reply files/en-us/web/api/htmlmediaelement/canplaytype/index.md]: OK, I'm going to make a separate PR for that.

**Review by dipikabh — APPROVED** [review]
Thanks a lot 👍

**Comment by hamishwillee** [comment]
Thanks. Great to get this in. Now I just have to make my review and writing habits match our agreed bullet list format :-)

**Pull request merged**